project.mml

```diff
@@ -1647,13 +1647,15 @@ Layer:
           COALESCE(
            'highway_' || CASE WHEN tags @> 'ford=>yes' OR tags @> 'ford=>stepping_stones' THEN 'ford' ELSE NULL END,
            'leisure_' || CASE WHEN leisure IN ('slipway', 'track') THEN leisure ELSE NULL END,
-           'attraction_' || CASE WHEN tags @> 'attraction=>water_slide' THEN 'water_slide' ELSE NULL END
+           'attraction_' || CASE WHEN tags @> 'attraction=>water_slide' THEN 'water_slide' ELSE NULL END,
+           'man_made_' || CASE WHEN man_made IN ('goods_conveyor') THEN man_made ELSE NULL END
```

Comment:
Rather than rendering this in the amenity-line layer, which will be on top of power-lines and many text features, please move this to the aerialways layer, like man_made=pipeline - this will render below bridges, power-lines, and text.

Reply:
fixed